tests/test_cli.py

```diff
@@ -41,6 +41,13 @@ def test_map_with_options(parser):
     assert args.reproduce == '/path'
     assert args.epoch == '123'
 
+def test_map_with_past_wait(capsys):
+    args = ['map', '-w', '1225411200']
+    with pytest.raises(SystemExit):
+        main(args)
+    captured = capsys.readouterr()
+    assert captured.err.startswith("usage:")
```

Comment:
Would prefer if you could make this more precise:

```suggestion
    assert "Cannot wait for a timestamp in the past (1225411200)" in captured.err
```

Reply:
We have this pattern with `captured.err.startswith("usage:")` in a few other places in the test, I would prefer if the actual error message could be identified in the check in these other places too but that can be further improved in a follow-up if you don't want to do it here (I can do it too). This should prevent that checks are not catching errors if a check becomes unreachable or is hidden behind another error.

Reply:
thanks yea i'm not sure why I left it like this. fixed other tests where appropriate below.